api/wd/mixins.md

```diff
@@ -53,9 +43,7 @@ In test files you can now start using `browser.login`:
 
 ```js
 // ...
-before(() => browser.login());
-
-before(() => browser.loadPage('/account'));
+before(() => browser.login().loadPage('/account'));
```

Comment:
Where is the `login` function defined if not in a mixins config?

Reply:
@aotarola  it is defined earlier in this markdown file